.github/workflows/precommit.yml

```diff
@@ -115,9 +115,19 @@ jobs:
           PYTEST_ADDOPTS: --color=yes --tb=short
           TOX_PARALLEL_NO_SPINNER: 1
         shell: bash
-      - if: ${{ matrix.python-version == '3.9' || matrix.python-version == '3.10' }}
-        name: Run tests (excluding udf and doctest tests)
-        run: python -m tox -e "py${PYTHON_VERSION/\./}-notudfdoctest"
+      - if: ${{ (matrix.python-version == '3.9' || matrix.python-version == '3.10') && contains('macos', matrix.os.download_name) }}
```

Comment:
Can we merge it with 3.8 part above?

Reply:
Right now we cannot because python 3.9/3.10 still requires `SNOWFLAKE_IS_PYTHON_RUNTIME_TEST` until we enable them in `setup.py`.